staticjinja/types.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+import typing as t
+from pathlib import Path  # noqa: F401
+
+import typing_extensions as te
```

Comment:
@killjoy1221 Is it possible to also place this in the `if t.TYPE_CHECKING:` so that we don't have to add `typing_extensions` as a runtime dependency?

Not the biggest deal if we have to add it as a runtime dep.

Reply:
Ha, I was about to do that anyway because readthedocs was complaining about it.

Reply:
Yeah I don't understand why RTD was freaking out? It also should be freaking out about the `watchdog` and the `docopt-ng` requirements? Whatever works lol